modules/vertx-swagger-router/src/main/java/io/vertx/ext/swagger/router/SwaggerRouter.java

```diff
@@ -29,6 +29,7 @@
 
     private static Logger VERTX_LOGGER = LoggerFactory.getLogger(SwaggerRouter.class);
 
+    private static Pattern PATH_PARAMETER_NAME = Pattern.compile("\\{(\\p{Alpha}[\\p{Alpha}\\d]*)\\}");
```

Comment:
I think the pattern is wring since the vertx definition is 
> Parameter names consist of any alphabetic character, numeric character or underscore.

I think your pattern should be `"\\{([\\p{Alpha}\\d_]+)\\}`